pkgs/top-level/all-packages.nix

```diff
@@ -7078,6 +7078,10 @@ with pkgs;
 
   k6 = callPackage ../development/tools/k6 { };
 
+  xk6 = callPackage ../development/tools/k6/xk6.nix {
+      go = go_1_17;
```

Comment:
why is go1.17 required? it is currently eol

Reply:
```suggestion
    go = go_1_17;
```